mapper/mapper_test.go

```diff
@@ -78,3 +78,19 @@ func TestDefaultPassThrough(t *testing.T) {
 
 	assert.JSONEq(t, string(expectedBody), string(actualBody), "json not equal")
 }
+
+func TestImplicitAnnotationsAreFiltered(t *testing.T) {
+
+	originalBody, err := ioutil.ReadFile("test_resources/implicit_annotations_v2.json")
+	if err != nil {
+		panic(err)
+	}
+	expectedBody, err := ioutil.ReadFile("test_resources/implicit_annotations_PAC.json")
+
+	if err != nil {
+		panic(err)
```

Comment:
And here.

Reply:
There is no "panic" in the code, not sure what you looked at.